docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
-# Updating an Application
```

Comment:
To update an application, there are 2 options, using the quick updates or using "Edit YAML". I do not see documentation about the second option, please include it.

Reply:
I added "Edit YAML" as the new step 3 in Updating an Application.

Reply:
I'd suggest rewording step 3 and step 4 to make it clearer that these 2 steps represent 2 different approaches to update an application. Something like:

Step3: To update an application by editting the YAML file directly, ...
Step4: To quickly update an application without having to touch the YAML file, ...

![image](https://user-images.githubusercontent.com/43203744/90619078-60075180-e243-11ea-9280-1dd87d4dd61c.png)

Reply:
Fixed